static/docs/user-guide/external-cache-and-outputs.md

```diff
@@ -14,7 +14,7 @@ You can take under DVC control files on an external storage with `dvc add` or
 specify external files as outputs for
 [DVC-files](/doc/user-guide/dvc-file-format) created by `dvc run` (stage files)
 DVC will track changes in those files and will reflect so in your pipeline
-[status](/doc/commands-reference/status). Currently, the following types of
+[status](/doc/command-reference/status). Currently, the following types of
```

Comment:
are we renaming it? it would be painful and I'm not sure it's needed

Reply:
Fixed back to `commands-reference` (in a few places).

But actually I do want to change it because the most used term is "Command Reference". Examples:

![image](https://user-images.githubusercontent.com/1477535/65822889-bf1e3b00-e219-11e9-8e51-81c31bfb13fe.png)
![image](https://user-images.githubusercontent.com/1477535/65822894-cc3b2a00-e219-11e9-9d89-24001f0e0b45.png)
![image](https://user-images.githubusercontent.com/1477535/65822898-da894600-e219-11e9-9b71-d6d793b54726.png)

![image](https://user-images.githubusercontent.com/1477535/65822902-ec6ae900-e219-11e9-8bf1-ffcc3fcc2329.png)

![image](https://user-images.githubusercontent.com/1477535/65822903-060c3080-e21a-11e9-8851-0747d40f048b.png)

Reply:
p.s. in fact I already have the change stashed on my branch but indeed it's pretty huge so I decided not to include it in this PR. Would need a separate one. Waiting for this one to merge first. Do you agree @shcheklein?

Reply:
Yep, let's change it if you see that it's more common. Please, do `git grep commands-reference` to double check that everything is handled.

Reply:
And, yes, it's better to do a separate PR 👍

Reply:
Done in 060c700.